docs/modules/ROOT/pages/explanation/system/idea.adoc

```diff
@@ -5,22 +5,43 @@ The system idea displays a high-level architecture of {product}.
 .{product} system idea
 image::system/systemidea.drawio.svg[]
 
-* An instance of Keycloak is responsible for authoritatively store data on:
-** Organizations
-** Teams
-** Users
-
-* Data owned by Keycloak will be replicated to each Zone.
-  It makes the cluster more resilient against connectivity issues between Keycloak and the Zone.
-
-* The policy engine https://kyverno.io/[Kyverno], installed on each Zone, is there to enforce policies that go beyond the capabilities of Kubernetes RBAC.
-  In the cases where Kyverno is not suitable, custom admission/mutating https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/[webhooks] will be engineered.
-
-* A controller instance on each Zone will further process the the local cache of entities coming from the Keycloak adapter.
-  It will be used to configure Kyverno for instance.
-
-* "APPUiO global" serves as control plane and is responsible for synchronizing mentioned entities to all APPUiO Zones.
-  It will run on one of the selected Zones only.
+Keycloak::
+An instance of Keycloak is responsible for authoritatively store data for:
++
+* Organizations
+* Teams
++
+The instance of Keycloak is not nessesarily authoritative for user identities.
+It's expected that most instances will setup user federation, identiy brokering or social logins where users are authenticated externally.
+That being the case, user sign up or registration is not part of core {product}.
+
+User and group sync::
+User and groups managed by Keycloak will be replicated to each {zone} using synchronization services running separately on each {zone}.
+It makes the cluster more resilient against connectivity issues between {idp} and an {zone}.
+
+Policy engine::
+A policy engine, also installed on each {zone}, is enforcing policies that go beyond the capabilities of Kubernetes RBAC.
+In the cases where a policy engine is not suitable, other solutions (for example https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/[admission webhooks]) are considered.
+
+REST API Endpoint::
```

Comment:
Do we have to be specific about the technology here? It could simply be called "API Endpoint".